packages/openapi-to-graphql/src/resolver_builder.ts

```diff
@@ -517,6 +781,9 @@ export function getResolver({
                   saneData = arraySaneData
                 }
 
+                // if (operation.callbacks)
```

Comment:
I was wondering if callbacks could be triggered in Mutation resolver.
Maybe through an option ?
Still it's probably safer to let them handled by user's backend ( and clearly easier ).